langchain/package.json

```diff
@@ -974,9 +974,9 @@
     "@elastic/elasticsearch": "^8.4.0",
     "@getmetal/metal-sdk": "*",
     "@getzep/zep-js": "^0.9.0",
-    "@gomomento/sdk": "^1.44.1",
-    "@gomomento/sdk-core": "^1.44.1",
-    "@gomomento/sdk-web": "^1.44.1",
+    "@gomomento/sdk": "^1.47.1",
+    "@gomomento/sdk-core": "^1.47.1",
+    "@gomomento/sdk-web": "^1.47.1",
```

Comment:
For the peer dependencies we need `@gomomento/sdk`, the Momento Node.js client; `@gomomento/sdk-web`, the Momento JavaScript Web client; and `@gomomento/sdk-core`, the shared messages and interface library. The last one is still needed as a peer dependency for the build to work, but users only need install `sdk` or `sdk-web`.